nixos/modules/services/x11/desktop-managers/gnome.md

```diff
@@ -132,6 +132,27 @@ Unfortunately, we lack a way for these to be managed in a completely declarative
 So you have to enable them manually with an Extensions application.
 It is possible to use a [GSettings override](#sec-gnome-gsettings-overrides) for this on `org.gnome.shell.enabled-extensions`, but that will only influence the default value.
 
+## Nautilus Extensions {#sec-gnome-nautilus-extensions}
+
+Nautilus extensions can add additional functionality to Nautilus, usually via the right-click menu. Some of these extensions are dynamically loaded from `$NAUTILUS_4_EXTENSIONS_DIR` environment variable, which is automatically set if [](#opt-services.gnome.core-utilities.enable) is true (enabled by default if GNOME is enabled).
```

Comment:
[Again](https://github.com/NixOS/nixpkgs/pull/240780#discussion_r1248203385), `NAUTILUS_4_EXTENSIONS_DIR` is an internal implementation detail, which should not be exposed in the user facing docs.